R/object_name_linters.R

```diff
@@ -1,3 +1,40 @@
+object_name_xpath <- paste0(
+  # assignments
+  "//SYMBOL[",
+  " not(preceding-sibling::OP-DOLLAR)",
```

Comment:
extract the repeated part as a variable and re-use it to make it easier to adjust going forward